ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/utils/Stubs.kt

```diff
@@ -0,0 +1,64 @@
+package com.revenuecat.purchases.ui.revenuecatui.utils
+
+import com.android.billingclient.api.ProductDetails
+import com.revenuecat.purchases.PresentedOfferingContext
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.InstallmentsInfo
+import com.revenuecat.purchases.models.Period
+import com.revenuecat.purchases.models.Price
+import com.revenuecat.purchases.models.PricingPhase
+import com.revenuecat.purchases.models.PurchasingData
+import com.revenuecat.purchases.models.SubscriptionOption
+import com.revenuecat.purchases.models.toRecurrenceMode
+
+@SuppressWarnings("LongParameterList")
+internal fun stubSubscriptionOption(
```

Comment:
Additionally, what do you think about naming this `previewSubscriptionOption`? And the file could also be something like `PreviewDataFactory` or something like that. I feel that's a better naming than `stub` so we don't confuse this with test code

Reply:
I am actually using `stubPricingPhase` in tests (`SubscriptionOptionExtensionsTest`) as well, to avoid duplication. What do you think?

Reply:
Right, I think it makes sense! But I still think it feels weird to add "stubs" to production code, which is why I would like to use the "preview" naming, which can still be used in tests I think.